components/site/src/lib.rs

```diff
@@ -855,6 +864,21 @@ impl Site {
         Ok(())
     }
 
+    pub fn generate_themes(&self) -> Result<()> {
+        ensure_directory_exists(&self.output_path)?;
+        for css_theme in &self.config.generate_theme_css {
+            println!("Generating CSS: {} for theme: {}", css_theme.file, css_theme.theme);
```

Comment:
No need to print.

Reply:
Good catch, was in for debugging purposes!